packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx

```diff
@@ -372,4 +372,24 @@ export class TreeViewWidget extends TreeWidget {
             }
         }
     }
+
+    protected render(): React.ReactNode {
+        return React.createElement('div', this.createContainerAttributes(), this.renderTreeInfo(), this.renderTree(this.model));
+    }
+
+    private _message: string | undefined;
```

Comment:
Why cannot be message be represented as first tree node with message as caption? then it will automatically get proper styling

Reply:
Can you point me to an example of this, if any. Maybe you right... I didn't tried to do this yet, but if it is possible to make such a node not to be selectable and disable any actions on it (like expand/open resource and so on), it may be a better solution.

Reply:
Well, the styling seems different in VS Code for the message, so if we want to align with that behaviour we need to keep it separate. Also, I guess we'd have to filter events to to that first item so the don't get passed to whoever might listen to them. I don't think it's saving work, in the end.

Reply:
ok